app/src/main/kotlin/com/aliucord/manager/installer/util/ManifestPatcher.kt

```diff
@@ -105,6 +115,7 @@ object ManifestPatcher {
                                 }
 
                                 override fun end() {
+                                    super.attr(ANDROID_NAMESPACE, REQUEST_LEGACY_EXTERNAL_STORAGE, -1, TYPE_INT_BOOLEAN, 1)
```

Comment:
Can you make this conditionally add if it hasnt been added yet like debuggable and useClearTextTraffic bellow

Reply:
Yeah, sure